tokenbalancehistory

bumbummen99 opened the pull request:
Added `tokenbalancehistory` action method to token.
This is a Pro endpoint and does require Pro API access.

bumbummen99 committed:
Add getHistoricAccountBalanceForTokenContractAddress to token

bumbummen99 committed:
Create CI.yml

bumbummen99 committed:
Add publish action

bumbummen99 committed:
Update token.ts

bumbummen99 committed:
Update query.ts

bumbummen99 committed:
Update token.ts

bumbummen99 committed:
Update token.ts

bumbummen99 committed:
Update query.ts

bumbummen99 committed:
Update Publish.yml

bumbummen99 committed:
Update Publish.yml

bumbummen99 committed:
Update Publish.yml

bumbummen99 committed:
Update package.json

bumbummen99 committed:
Delete .github/workflows directory

bumbummen99 committed:
Update package.json

jpgarcia commented on src/token.ts:
```diff
@@ -39,8 +39,28 @@ function getAccountBalanceForTokenContractAddress(
   )
 }
 
+function getHistoricAccountBalanceForTokenContractAddress(
```
I'd rename this as `getHistoricialAccountBalanceForTokenContractAddress` to be consistent with the docs

bumbummen99 committed:
Update token.ts

jpgarcia commented on src/token.ts:
```diff
@@ -39,7 +39,7 @@ function getAccountBalanceForTokenContractAddress(
 }
 
-function getHistoricAccountBalanceForTokenContractAddress(
+function getHistoricialAccountBalanceForTokenContractAddress (
```
It seems you missed the rename in the `export default` section

jpgarcia replied:
While reviewing it I also noticed that I misspelled the `Historicial` word!! it should be `getHistoricalAccountBalanceForTokenContractAddress`

Sorry about that!

bumbummen99 replied:
No problem :) Fixing it rn

bumbummen99 committed:
Fix

Rename new method to `getHistoricalAccountBalanceForTokenContractAddress`,  
Fix new method not being exported,

jpgarcia commented:
:tada: This PR is included in version 1.2.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/bsc-scan/v/1.2.0)
- [GitHub release](https://github.com/jpgarcia/bsc-scan/releases/tag/v1.2.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
